src/index.html

```diff
@@ -11,8 +11,94 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Link to a SCSS file should be replaced with a compiled CSS file link. Browsers do not understand SCSS syntax, so it must be compiled to CSS before being linked in the HTML.